source/MRMesh/MRMesh.vcxproj.filters

```diff
@@ -1805,9 +1808,6 @@
     <ClCompile Include="MRICP.cpp">
       <Filter>Source Files\MeshAlgorithm</Filter>
     </ClCompile>
-    <ClCompile Include="MRMeshTrimWithPlane.cpp">
```

Comment:
Please restore MRMeshTrimWithPlane.h and MRMeshTrimWithPlane.cpp exactly, and put new functions in PointCloud or in new file.